src/server/lib/idapi/auth.ts

```diff
@@ -45,13 +58,34 @@ export const read = async (
   };
   try {
     return responseToEntity(
-      (await idapiFetch(AUTH_URL, {
+      (await idapiFetch('/auth/redirect', {
```

Comment:
I inlined this to reduce the need for scrolling

Reply:
I wonder if there is a need for an enum of Identity URLs, similar to what we have for the internal routes in `shared/model/Routes.ts`. But this isn't a huge issue.

Reply:
I actually find these kind of abstractions make it harder to read the code. I much prefer inlined, hardcoded values because it means I see right away what the value it and don't need to click around opening up new files to work it out, especially when the value is only ever used once.

Granted though, this is a subjective area!

Reply:
That's fair. Considering that the IDAPI endpoints are only ever specific to the route we're calling there's no real need for the abstraction, and they're only every usually being use in a single place.